arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -353,20 +365,42 @@
       }
     },
     "Registry": {
-      "description": "An object that represents a container registry.",
       "allOf": [
         {
           "$ref": "#/definitions/Resource"
         }
       ],
       "properties": {
+        "sku": {
+          "$ref": "#/definitions/Sku"
+        },
         "properties": {
           "$ref": "#/definitions/RegistryProperties",
           "description": "The properties of the container registry.",
           "x-ms-client-flatten": true
         }
       }
     },
+    "Sku": {
+      "properties": {
+        "name": {
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "type": "string"

```

Comment:
Change "type": "string" to
          "x-ms-enum": {
            "name": "SkuName",
            "modelAsString": false
          }